lib/graphql/transaction.ts

```diff
@@ -204,6 +222,17 @@ export class Transaction extends IterableModel<TransactionModel> {
     return result.categorizeTransaction;
   }
 
+  /**
+   * Updates a transaction
+   *
+   * @param args   query parameters including category, userSelectedBookingDate and personalNote
+   * @returns      the transaction with updated categorization data and updated personalNote
+   */
+  public async updateMeta(args: MutationUpdateTransactionArgs) {
```

Comment:
shouldn't it be called updateTransaction? what do you think?

Reply:
I would think just `update`, because `updateTransaction` would be used as `models.transaction.updateTransaction(...)` which makes "Transaction" redundant.

We also have the same approach in general for model methods, e.g. `models.transaction.categorize`, `models.transfer.createOne`, `models.transfer.update`... It's not always consistent but recently we have tried to stick to the convention of not repeating model name in method name.

Between `transaction.updateMeta` and `transaction.update` I would also have a slight preference for `transaction.update` (because it would allow more generic updates than just meta, if we have the requirement later), but I wouldn't object in other name.

Reply:
Thanks for sharing your thoughts about the new method name. I'll go for `update()` as this follows the new convention of  "not repeating model name in the method name", so we stay consistent.